practice_8/practice_8/Motocycle.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace practice_10
+{
+    internal sealed class Motocycle
+    {
+        int Id;
+        public string producer;
+        public string model;
+        public double numberKm;
+        public DateTime Date;
```

Comment:
we need to define all these as C# Property, for instance
`public string Model{get; set;}`